### DISCUSSION TITLE
Regarding Namespace @prefix lbd: <https://linkebuildingdata.org/LBD#>

### DISCUSSION THREAD

**Discussion by ajayka7** [jyrkioraskari/IFCtoLBD#81] [Q&A]
Hii, 

I am a researcher in the area of Construction Management, and I was trying to understand how the IFCtoLBD converter works. Using the tool, I used a simple IFC file exported from Revit and converted it to LBD. The namespaces in the converted file looked like something below:
@prefix owl: <http://www.w3.org/2002/07/owl#> .
@prefix bot: <https://w3id.org/bot#> .
@prefix ifc: <https://standards.buildingsmart.org/IFC/DEV/IFC2x3/TC1/OWL#> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
@prefix lbd: <https://linkebuildingdata.org/LBD#> .
@prefix props: <http://lbd.arch.rwth-aachen.de/props#> .
@prefix geo: <http://www.opengis.net/ont/geosparql#> .
@prefix unit: <http://qudt.org/vocab/unit/> .
@prefix IFC4-PSD: <https://www.linkedbuildingdata.net/IFC4-PSD#> .
@prefix smls: <https://w3id.org/def/smls-owl#> .
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix inst: <https://www.ugent.be/myAwesomeFirstBIMProject#> .
@prefix fog: <https://w3id.org/fog#> .

On reading the namespaces, I could not figure out what _@prefix lbd: <https://linkebuildingdata.org/LBD#>_ is, i.e., which ontology is it? On clicking the link given in the brackets, it leads to nowhere. (it says the DNS address can not be found). As a lot of data in the converted file is represented using the lbd prefix, it became a bit difficult for me to understand this. 
Could anyone please help? I want to know what exactly the lbd prefix points to and what all classes, properties etc. it has. Sorry, if it is a silly question. I am kind of new to this area, so trying to figure out the things. Thanks.

Best Regards,
Ajay

**Comment by jyrkioraskari** [comment]
Hi  Ajay,
You are right. It is an oldaddress.  I will figure out how this can be fixed.

J.
